sw/device/tests/crypto/BUILD

```diff
@@ -28,10 +28,11 @@ opentitan_functest(
 autogen_cryptotest_header(
     name = "ecdsa_p256_verify_testvectors_hardcoded_header",
     hjson = "//sw/device/tests/crypto/testvectors:ecdsa_p256_verify_testvectors_hardcoded",
-    template = "ecdsa_p256_verify_testvectors.h.tpl",
-    test_setter = "ecdsa_p256_verify_set_testvectors.py",
+    template = ":ecdsa_p256_verify_testvectors.h.tpl",
+    tool = "//sw/device/tests/crypto:ecdsa_p256_verify_set_testvectors",
     deps = [
         requirement("hjson"),
+        requirement("mako"),
```

Comment:
since these are dependencies of the `py_binary` `tool` do you need these deps in the instantiation of this custom rule? i.e., what happens if you remove the "deps" attributed completely here? (if this works, applies throughout)

Reply:
Thanks! I will remove them.